.github/workflows/nightly.yml

```diff
@@ -77,7 +77,9 @@ jobs:
           build-args: |
             ROS_DISTRO=${{ matrix.ros_distribution }}
             BASE_IMAGE=ghcr.io/${{ github.repository }}/minimal-rmf:${{ matrix.ros_distribution }}
-          tags: ghcr.io/${{ github.repository }}/api-server:${{ matrix.ros_distribution }}
+          tags: |
+            ghcr.io/${{ github.repository }}/api-server:${{ matrix.ros_distribution }}
+            ${{ matrix.ros_distribution == 'jazzy' && format('ghcr.io/{0}/api-server:latest', github.repository) || null}}
```

Comment:
I just realized that this is a nightly ci, so we should tag them as `jazzy-nightly` and `rolling-nightly` instead. `jazzy` and `rolling` should point to the latest released version.